cmd/maya-apiserver/app/command/start.go

```diff
@@ -126,6 +129,22 @@ func NewCmdStart() *cobra.Command {
 	return cmd
 }
 
+func performPreInflightChecks() error {
+	return checkForNDMrelatedCRDs()
+}
+
+func checkForNDMrelatedCRDs() error {
+	_, err := bdc.NewKubeClient().List(metav1.ListOptions{})
```

Comment:
Instead of `CR` we need to check for `CRD`

Reply:
Ignore above comment

Reply:
doable.. its just another way.. followed from pool-mgmt code

Reply:
Yeah after looking into it I also resolved the comment